### PR TITLE
Docs update for tinacms/tinacms#1704

### PR DESCRIPTION
**Pull request by austincondiff**
Adds documentation for the radio group field in tinacms/tinacms#1704.

__Note:__ I was unable to test this because I do not have an `.env` with GitHub client id and secret so my dev server will not start. I did not find documentation explaining how to set this up.

### CODE REVIEW

**Review by DirtyF — COMMENTED** [review]
Thanks @austincondiff this is awesome 🤩 

From a quick first look at the deploy preview https://tinacms-site-n-git-fork-austincondiff-docs-fields-radi-30e617.tinacms.vercel.app/docs/plugins/fields/radio-group/ 

- [ ] radio-field is not displayed in the sidebar navigation
- [x] ~~images do not appear~~

**Comment by DirtyF** [comment]
@austincondiff Do https://tina.io/guides/nextjs/github/github-oauth-app/ help to set up local env variables?

File: content/docs/plugins/fields/radio-group.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+consumes:
+  - file: /packages/@tinacms/fields/src/RadioGroup.tsx
+    details: Shows radio group field and Option interfaces
+  - file: /packages/@tinacms/fields/src/plugins/RadioGroupFieldPlugin.tsx
+    details: Shows radio group field and Option interfaces
+---
+
+The `radio-group` field represents a group of radio inputs. There are two variants that can be used, "radio" and "button", and can rendered in a "horizontal" or "vertical" direction. The "radio" variant in a "horizontal" direction is used by default.
+
+![TinaCMS Radio Group Field using a "radio" variant in a "horizontal" direction](/img/fields/radio-group-field-horizontal-radio.png)
```
Comment (DirtyF) [inline content/docs/plugins/fields/radio-group.md:14]:
```suggestion
![TinaCMS Radio Group Field using a "radio" variant in a "horizontal" direction](/img/fields/radio-group-field-horizontal-radio.gif)
```

File: content/docs/plugins/fields/radio-group.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+The "radio" variant can be rendered in a vertical direction like so.
+
+![TinaCMS Radio Group Field using a "radio" variant in a "vertical" direction](/img/fields/radio-group-field-vertical-radio.png)
```
Comment (DirtyF) [inline content/docs/plugins/fields/radio-group.md:18]:
```suggestion
![TinaCMS Radio Group Field using a "radio" variant in a "vertical" direction](/img/fields/radio-group-field-vertical-radio.gif)
```

File: content/docs/plugins/fields/radio-group.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+This is what the radio group "button" variant will look like.
+
+![TinaCMS Radio Group Field using a "button" variant in a "horizontal" direction](/img/fields/radio-group-field-horizontal-button.png)
```
Comment (DirtyF) [inline content/docs/plugins/fields/radio-group.md:22]:
```suggestion
![TinaCMS Radio Group Field using a "button" variant in a "horizontal" direction](/img/fields/radio-group-field-horizontal-button.gif)
```

File: content/docs/plugins/fields/radio-group.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+The "button" variant" can also have a "vertical" direction as well.
+
+![TinaCMS Radio Group Field using a "button" variant in a "vertical" direction](/img/fields/radio-group-field-vertical-button.png)
```
Comment (DirtyF) [inline content/docs/plugins/fields/radio-group.md:26]:
```suggestion
![TinaCMS Radio Group Field using a "button" variant in a "vertical" direction](/img/fields/radio-group-field-vertical-button.gif)
```

**Review by DirtyF — CHANGES_REQUESTED** [review]
@austincondiff for the page to be linked in the sidebar nav, you need to add manually an entry in the `content/toc-doc.json` line 141 between `toggle` and `select` fields: 

```json
                    {
                        "id": "/docs/plugins/fields/radio-group",
                        "slug": "/docs/plugins/fields/radio-group",
                        "title": "Radio Group"
                    },
```

**Comment by austincondiff** [comment]
The sidebar entry should be there now. ✅

We may at some point may look into making the sidebar editable by the cms and automating the prev/next links so that doesn't have to be done manually (unless we want more control by not doing so).

**Review by DirtyF — APPROVED** [review]

**Pull request merged**